packages/client/hmi-client/src/components/widgets/slider-panel.vue

```diff
@@ -11,7 +11,11 @@
 					:class="`slider-header-item pi ${directionMap[direction].iconOpen}`"
 					@click="emit('update:isOpen', false)"
 				/>
-				<h4 class="slider-header-item">{{ header }}</h4>
+				<slot name="header"></slot>
+				<div class="column">
```

Comment:
Please feel free to give me a better class name than "column" if you have any

Reply:
I mean you can just make it a `<section>` and in then 
```html
<style>
   ...
   section { ... } /* instead of .column */
   ...
</style>
```